config/content.d/developer.rackspace.com.json

```diff
@@ -31,11 +31,11 @@
           "/docs/cloud-feeds/v1/developer-guide/": "https://github.com/rackerlabs/standard-usage-schemas",
           "/docs/dedicated-load-balancers/v2/": "https://github.com/rackerlabs/docs-dedicated-networking/",
           "/docs/glossary/": "https://github.com/rackerlabs/docs-rackspace/",
-          "/docs/private-cloud/release-notes/v13": "https://github.com/rcbops/rpc-openstack/v13/",
+          "/docs/private-cloud/rpc/v14/": "https://github.com/rackerlabs/docs-rpc/v14/",
           "/docs/private-cloud/rpc/v13/": "https://github.com/rackerlabs/docs-rpc/v13/",
           "/docs/private-cloud/rpc/v12/": "https://github.com/rackerlabs/docs-rpc/v12/",
-          "/docs/private-cloud/rpc/v11": "https://github.com/rackerlabs/docs-rpc/v11/",
-          "/docs/private-cloud/rpc/v10": "https://github.com/rackerlabs/docs-rpc/v10/",
+          "/docs/private-cloud/release-notes/v14": "https://github.com/rcbops/rpc-openstack/v14/",
```

Comment:
I have yet to go through and figure out the exact mechanism here, but worst comes to worst, we get a blank page. I don't think it will break production. We could try it on a local deconst if you want (essentially, staging for nexus-control).